cinema/management/commands/initadmin.py

```diff
@@ -0,0 +1,20 @@
+from typing import Any
+
+from django.conf import settings
+from django.core.management import BaseCommand
+from user.models import User
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args: Any, **options: dict[str]) -> None:
+        if not User.objects.exists():
```

Comment:
You need filter by email because for example:
1. I do python manage.py migrate
2. python manage.py createsuperuser
3. python manage.py initadmin
As a result i do not have admin with settings.ADMINS["EMAIL"]

Reply:
Not yet fixed do as follow:
if not User.objects.filter(email=settings.ADMINS["EMAIL"]).exists():